book/version_control.md

```diff
@@ -155,53 +168,40 @@ Further paragraphs come after blank lines.
 - Use a hanging indent
 ```
 
-Branches are independent copies of a project's history, copied from the state of the parent branch at a specific point in that branches history.
-They help to support multiple changes to a project that are developed in parallel.
-The `master` is the default name for the original branch of a repository, however, some platforms are changing this default to `main`.
+A concise summary of a your change is usually sufficient, but remember that it is these commit messages that will be used in the future to understand what changes have been made to your project. You might rely on these to identify where an error has been introduced, so it is important that you write these messages clearly and informatively.
```

Comment:
Maybe add something as an example: 

Someone looking to understand why a particularly cryptic and undocumented piece of code has been introduced might seek clarity by finding what commit introduces the change and seeing what the author was trying to address with that particular commit in its message.